lms/static/scripts/frontend_apps/test/index-test.js

```diff
@@ -89,6 +89,20 @@ describe('LMS frontend entry', () => {
     });
   });
 
+  it('console logs debug values', () => {
+    const log = sinon.spy(console, 'log');
```

Comment:
This console method won't be restored if the test fails. You can fix this with a finally block:

```js
const log = sinon.stub(console, 'log');

try {
  // Test setup, asserts etc.
} finally {
  log.restore();
}
```

Also, you'll want to use `sinon.stub` here rather than `spy`, to prevent the spy from executing the real `console.log` and producing logging noise from the test runner.